### PR TITLE
Update extensionsGallery.json

### PR DESCRIPTION
**Pull request by jordan-pieces**
Added pieces azure data studio plugin - code snippets + copilot in azure data studio. 

There should be no details here - just to note that we used GCP for our assets as we hope that is not an issue.

### CODE REVIEW

**Comment by jordan-pieces** [comment]
@microsoft-github-policy-service agree company="Pieces.app"

File: extensionsGallery.json [hunk 1]
```diff
@@ -5331,7 +5331,66 @@
 				],
 				"statistics": [],
 				"flags": ""
-			}
+			},
+
+			{
+					"extensionId": "106",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:5337]: Please also: 

1. Update the insiders gallery file
2. Update the ResultCount value at the end of each file (increase it by 1)

Comment (jordan-pieces) [reply extensionsGallery.json:5337]: making these updates now

Comment (jordan-pieces) [reply extensionsGallery.json:5336]: @Charles-Gagnon should now be updated

**Comment by Charles-Gagnon** [comment]
Let me check about using GCP for the static assets - I think it should be fine though.

**Comment by Charles-Gagnon** [comment]
Unfortunately not - we have the origin set to `vscode-file://vscode-app` for our extension gallery requests which means it's blocked by CORS unless the GCP endpoint specifically allows it. And given that we can't control that and would have to have everyone using it to make that change themselves that's not something we can do at this time.

Do you have a Github repo you could upload these assets to? If not then we can host them ourselves - it'll just mean that you would have to reach out to us to make any changes in the future.

**Comment by jordan-pieces** [comment]
i can host them on our docs.pieces.app site actually, thanks for that info
there. let me move these to the new location and i will update the links

Sent via Superhuman iOS ***@***.***>

**Comment by jordan-pieces** [comment]
@Charles-Gagnon updated with new public url paths 👍

**Comment by jordan-pieces** [comment]
@microsoft-github-policy-service agree company="Pieces.app"

**Comment by Charles-Gagnon** [comment]
Thank you for your contribution! You should see the gallery updated in a few minutes.

**Pull request merged**